### PR TITLE
estimates vs quotes

### PR DESCRIPTION
**Pull request by aljazceru**
- Added support for estimates vs quotes, expanded the examples
- Added support for both scenarions:
1) client wants to purchase X bitcoin
2) client wants to spend X fiat

### CODE REVIEW

File: FLS01/README.md [hunk 1]
```diff
@@ -130,10 +135,19 @@ POST /quote
...
+  "amount_btc": 5000000 # sats
+  "currency_id":1,
+  "payment_option_id":1
+}
 ```
 
-- `amount_fiat` what the client wants to spend
-    - must be greater than 0 and less than 1000
+- `amount_fiat` (optional) amount of fiat the client wants to spend (unit cents)
+- `amount_sats` (optional) amount of bitcoin the client wants to purchase (unit sats)
```
Comment (ademcan) [inline FLS01/README.md:150]: That's a great add-on

File: FLS01/README.md [hunk 2]
```diff
@@ -246,18 +273,22 @@ Response:
...
     "payment_option_id":1,
     "amount_sats" : 800000 ,
+    "btc_price": 69420
+    "order_fee": 1234
     "order_status": "finished"
     "order_status_date": "2023-09-20T00:25:11.123Z"
   },
     "order_id": "8ed13c2a-a8c6-4f0e-b43e-3fdbf1f0333a": {
-    "amount_fiat": "1000",
+    "amount_fiat": 1000,
```
Comment (ademcan) [inline FLS01/README.md:286]: add # unit cents ?

Comment (aljazceru) [reply FLS01/README.md:286]: those comments are more for added info, its specified at the top what the units are, but sure i can add for clarity

Comment (ademcan) [reply FLS01/README.md:286]: Right, didn't check the whole file again, all good then!

Comment (aljazceru) [reply FLS01/README.md]: @ademcan i've added units to everything now

**Pull request merged**